gr-fosphor.rb

```diff
@@ -17,6 +17,7 @@ def install
       args = %W[
         -DCMAKE_SHARED_LINKER_FLAGS='-Wl,-undefined,dynamic_lookup'
         -DPYTHON_LIBRARY='#{HOMEBREW_PREFIX}/lib/libgnuradio-runtime.dylib'
+        -DFREETYPE2_INCLUDE_DIR_ftheader='#{HOMEBREW_PREFIX}/Cellar/include'
```

Comment:
I just double checked, and you're correct. I honestly know very little about homebrew. I held off on switching to it from macports specifically because macports has easier GNU Radio installation.

This bug is weird. I'm wondering if it's a problem with gr-fosphor's cmakelists.txt. It seem defining `FREETYPE2_INCLUDE_DIR_ftheader` as anything will let it build.